docs/src/main/sphinx/release/release-431.md

```diff
@@ -0,0 +1,58 @@
+# Release 431 (25 Oct 2023)
+
+## General
+
+* Add support for [](/routines). ({issue}`19308`)
+* Add support for [](/sql/create-function) and [](/sql/drop-function) statements. ({issue}`19308`)
+* Add support for the `REPLACE` modifier to the `CREATE TABLE` statement. ({issue}`13180`)
+* Disallow a `null` offset for the {func}`lead` and {func}`lag` functions. ({issue}`19003`)
+* Improve performance of queries with short running splits. ({issue}`19487`)
+
+## Security
+
+* Support defining rules for procedures in file-based access control. ({issue}`19416`)
+* Mask additional sensitive values in log files. ({issue}`19519`)
+
+## JDBC driver
+
+* Add a new JDBC property `legacyPreparedStatements` that controls whether
+  `EXECUTE IMMEDIATE` is used to execute prepared statements. ({issue}`18957`)
+
+## Delta Lake connector
+
+* Replace the `hive.metastore-timeout` Hive metastore configuration property
+  with the `hive.metastore.thrift.client.connect-timeout` and
+  `hive.metastore.thrift.client.read-timeout` properties. ({issue}`19390`)
+
+## Hive connector
+
+* Add support for [SQL routine management](sql-routine-management). ({issue}`19308`)
+* Replace the `hive.metastore-timeout` Hive metastore configuration property
+  with the `hive.metastore.thrift.client.connect-timeout` and
+  `hive.metastore.thrift.client.read-timeout` properties. ({issue}`19390`)
+* Improve support for concurrent table statistics updates in Glue. ({issue}`19463`)
```

Comment:
fyi @findepi and @findinpath .. I reworded to this. Also does this apply to Iceberg, Delta Lake, or Hudi connectors as well?

Reply:
To my knowledge it applies only to Hive.

Reply:
Thanks